lib/hex.ml

```diff
@@ -35,7 +45,16 @@ let to_char x y =
   in
   Char.chr (code x lsl 4 + code y)
 
-let of_helper ?(ignore=[]) (next : int -> char) len =
+let of_string_fast s =
+  let len = String.length s in
+  let buf = Bytes.create (len * 2) in
+  for i = 0 to len - 1 do
+    Bytes.set buf       (i * 2)  hexa1.[Char.code s.[i]];
+    Bytes.set buf (succ (i * 2)) hexa2.[Char.code s.[i]];
```

Comment:
For style or for performance?  [Char.code](https://github.com/ocaml/ocaml/blob/c59bce11a5528d78d2177660a14d7f0a489b1407/stdlib/char.mli#L16) is defined as `"%identity"`, so it doesn't cost anything.